src/js/components/AppListItemComponent.jsx

```diff
@@ -47,43 +48,75 @@ var AppComponent = React.createClass({
   },
 
   onClick: function () {
-    this.context.router
-      .transitionTo("app", {appId: encodeURIComponent(this.props.model.id)});
+    var model = this.props.model;
+    var router = this.context.router;
+    if (model.isGroup) {
+      router.transitionTo("group", {groupId: encodeURIComponent(model.id)});
+    } else {
+      router.transitionTo("app", {appId: encodeURIComponent(model.id)});
+    }
+  },
+
+  getHealthBar: function () {
+    var model = this.props.model;
+    return (
+      <td className="text-right health-bar-column">
+        <AppHealthComponent model={model} />
+      </td>
+    );
+  },
+
+  getStatus: function () {
+    var model = this.props.model;
+    return (
+      <td className="text-right status">
+        <AppStatusComponent model={model} />
+      </td>
+    );
   },
 
   render: function () {
     var model = this.props.model;
+    var status = null;
+    var healthBar = null;
+    var colSpan = 3;
+    var className = "group";
+    var name = ViewHelper.getRelativePath(model.id, this.props.currentGroup);
```

Comment:
getRelativePath is used only once here, so please define it as a "private" function on top of AppListItemComponent.jsx. 
Then we can get rid of this ViewHelper-file, which isn't the right terminology in react.